persistence/persistence-spark/src/test/java/org/cbioportal/persistence/spark/DiscreteCopyNumberSparkRepositoryTest.java

```diff
@@ -0,0 +1,100 @@
+package org.cbioportal.persistence.spark;
+
+import org.apache.spark.sql.*;
+import org.cbioportal.model.CopyNumberCountByGene;
+import org.cbioportal.model.Gene;
+import org.cbioportal.model.MolecularProfile;
+import org.cbioportal.persistence.GeneRepository;
+import org.cbioportal.persistence.MolecularProfileRepository;
+import org.cbioportal.persistence.spark.util.ParquetLoader;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import scala.collection.Seq;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+@ContextConfiguration("/testSparkContext.xml")
+@TestPropertySource("/testPortal.properties")
+@Configurable
+public class DiscreteCopyNumberSparkRepositoryTest {
+
+    @Mock
+    private SparkSession spark;
+
+    @Mock
+    private MolecularProfileRepository molecularProfileRepository;
+
+    @Mock
+    private GeneRepository geneRepository;
+
+    @Mock
+    private ParquetLoader parquetLoader;
+    
+    @InjectMocks
+    private DiscreteCopyNumberSparkRepository discreteCopyNumberSparkRepository;
+
+    private Dataset<Row> ds;
+
+    @Before
+    public void setup() {
+        ds = mock(Dataset.class);
+        when(parquetLoader.loadStudyFiles(any(SparkSession.class), anySet(), anyString(), anyBoolean())).thenReturn(ds);
+        when(parquetLoader.loadCaseListFiles(any(SparkSession.class), anySet(), anyBoolean())).thenReturn(ds);
```

Comment:
instead of mocking can we have some test parquet files?

Reply:
I added some parquet files and made this an integration test. also cleaned up other junit tests files.